src/ontology/components/mappings.owl

```diff
@@ -142,6 +142,9 @@
   <Class rdf:about="http://purl.obolibrary.org/obo/CL_0000381">
     <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">FBbt:00005130</oboInOwl:hasDbXref>
   </Class>
+  <Class rdf:about="http://purl.obolibrary.org/obo/CL_0000850">
+    <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">FBbt:00005133</oboInOwl:hasDbXref>
```

Comment:
@gouttegd - could you help make sure this is correct - I think it got updated with import updates

Reply:
That’s correct, yes. It’s a change that I made recently in the FBbt-to-CL mappings. That it got picked up during an import update is the expected behaviour, I believe.